### PR TITLE
Apply style patch

### PR DESCRIPTION
**Pull request by Quezler**
PSR 2

### CODE REVIEW

**Comment by LukeTowers** [comment]
Changes simply for the sake of code formatting will not be accepted. Code formatting changes may be accepted within the same file when other functional changes are being applied, but not by themselves.

**Comment by daftspunk** [comment]
We prefer to keep robots away from the code, thanks though @Quezler!

**Comment by Quezler** [comment]
@daftspunk actually it was not a bot, it was me running php-cs-fixer myself.

Since the contribution guide stated PSR-2, i thought it would be a good idea to run it against the current code base.

(I personally see no valid reason not to merge this though, sounds a bit counterintuitive)

**Comment by daftspunk** [comment]
Wait, so you made changes to 223 files by hand? If so, that's huge, please put your hand up before making this kind of effort!

There is one thing you could have missed, there are a few exceptions to PSR 2 that are not considered in the changes:
http://octobercms.com/help/guidelines/developer#psr-exceptions

**Comment by Quezler** [comment]
I kinda followed the contribution guidelines at the bottom of https://github.com/octobercms/october, i was unaware of the link you had sent 😲 

And well yea, i like using octobercms, and when i develop plugins i tend have my project directory in PSR-2, and core files not in PSR-2 will eventually conflict when i do updates 💣

**Comment by daftspunk** [comment]
Bummer dude. Not really sure what you mean by conflict? If you can fix all the if+catch statements we could take another look.

**Comment by Quezler** [comment]
@daftspunk allright, i'll take a look into it, i shall ping you in here when its done 😄

**Comment by Quezler** [comment]
@daftspunk hmm, apparently since this pull request is closed my commit didn't get added here, could you reopen it or should i make a new pr?

File: modules/cms/classes/CodeParser.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ class CodeParser
     /**
      * @var mixed The internal cache, keeps parsed object information during a request.
      */
-    static protected $cache = [];
+    protected static $cache = [];
```
Comment (daftspunk) [inline modules/cms/classes/CodeParser.php]: LOL 

what?

Comment (Quezler) [inline modules/cms/classes/CodeParser.php]: I didn't make up the psr 2 rules 😏

**Comment by Quezler** [comment]
👍

**Comment by daftspunk** [comment]
Looks good, thanks!

**Comment by Quezler** [comment]
@daftspunk Glad to have contributed to octobercms!🎖

**Comment by Quezler** [comment]
(thats odd, shoudn't i be marked by github as a contributer? since this was merged?)

File: modules/cms/lang/es/lang.php [hunk 2]
```diff
@@ -10,15 +10,15 @@
         'invalid_file_extension' => 'Extensión de archivo inválida: :invalid. Las extensiones permitidas son: :allowed.',
         'error_deleting' => 'Error borrando el archivo template ":name". Por favor, revisa los permisos de escritura.',
         'delete_success' => 'Los templates fueron borrados satisfactoriamente: :count.',
-	'file_name_required' => 'Falta el nombre del campo del archivo.',
-	'safe_mode_enabled' => 'Modo seguro esta actualmente activado.'
+    'file_name_required' => 'Falta el nombre del campo del archivo.',
```
Comment (LukeTowers) [inline modules/cms/lang/es/lang.php]: Why was this merged @daftspunk? This isn't even consistent reformatting, it looks automated to me.

**Comment by LukeTowers** [comment]
@Quezler You aren't considered a contributor by Github until your commits make it to master. It also doesn't count PRs that have had to be squashed and merged (i.e. PRs with unnecessarily cluttered history)

**Pull request merged**